### DISCUSSION TITLE
Any chance of a G-Helper but for Alienware laptops?

### DISCUSSION THREAD

**Discussion by JimmyXD2** [seerge/g-helper#3243] [General]
Any chance of a G-Helper but for Alienware laptops? The AWCC is very heavy and only gives me errors, there is a third party program but I feel that it is not as good as I would like, seeing your G-Helper program I wanted to try it right away but I realized that it only works for Asus laptops

**Comment by seerge** [comment]
@JimmyXD2 hello, G-Helper is built around proprietary Asus interface (for pretty much everything, like modes, fan settings, GPU toggling, etc). Those settings simply don't exist on devices from other brands :)

**Comment by juliothomaz** [comment]
> Any chance of a G-Helper but for Alienware laptops? The AWCC is very heavy and only gives me errors, there is a third party program but I feel that it is not as good as I would like, seeing your G-Helper program I wanted to try it right away but I realized that it only works for Asus laptops

I really would love it... My Desktop is a R9 5950x on TUF that runs like a charm with G-Helper. My Laptop is an Alienware R9 which I woul love to set G-Helper running in it... 

Kind Regards...
